draft-ietf-add-svcb-dns.md

```diff
@@ -47,13 +47,15 @@ when, and only when, they appear in all capitals, as shown here.
 
 # Name form
 
-Names are formed using Port-Prefix Naming ({{Section 2.3 of SVCB}}), with a scheme of "dns".  For example, SVCB records for a DNS service identified as "`dns1.example.com`" would be located at "`_dns.dns1.example.com`".
+In this specification, DNS services are identified by a "binding authority" (which MUST be an "`authority`" as in {{Section 3.2 of !RFC3986}}) and an "authentication name" (which MUST be a "`host`" as in {{Section 3.2.2 of !RFC3986}})).  Unless otherwise specified, the authentication name is the Host from the binding authority.
```

Comment:
I had a hard time trying to understand this formulation.  I understand that the goal is to provide unambiguous definitions for key concepts, but this requires a bunch of background knowledge.

The next paragraph is much easier to understand.  Consider following that with a clearer definition and explanation of what binding authority is (this reduces to name plus port, assuming that you don't want userinfo as well)

Including an explanation of why you might choose to authenticate the server using a different name is necessary to motivate the inclusion of an "authentication name" as a concept.